src/main/java/com/thealgorithms/maths/SquareRootWithNewtonRaphsonMethod.java

```diff
@@ -0,0 +1,48 @@
+package com.thealgorithms.maths;
+
+import java.util.Scanner;
+
+/*
+*To learn about the method, visit the link below :
+* https://en.wikipedia.org/wiki/Newton%27s_method
+*
+* To obtain the square root, no built-in functions should be used
+*
+* The formula to calculate the root is : root = 0.5(x + n/x),
+* here, n is the no. whose square root has to be calculated and
+* x has to be guessed such that, the calculation should result into
+* the square root of n.
+* And the root will be obtained when the error < 0.5 or the precision value can also
+* be changed according to the user preference.
+*/
+
+public class SquareRootWithNewtonRaphsonMethod {
+
+    public static void main(String[] args) {
+
+        Scanner in = new Scanner(System.in);
+        System.out.print("Enter n to find it's square root: ");
+        int n = in.nextInt();
+
+        System.out.println("Square root of " + n + " is " + sqRoot(n));
+    }
```

Comment:
Please, remove `main()` and add JUnit tests instead

Reply:
Hey Siriak, I'm actually new to it and I don't know about JUnit. I tried to find some helpful tutorials, but they didn't work out for me. What shall I do?

Reply:
See examples in `src/tests` directory

Reply:
Added JUnit tests.